bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -12,6 +12,8 @@
   {% javascript 'firefox_new_firstrun' %}
 {% endblock %}
 
+{% block global_nav %}{% endblock %}
+
```

Comment:
I thought the same thing, but this template is only shown for f=99 and f=100. So I'm assuming they just mean the FxA UI that was shown in that past experiment.